pkg/controller/federatedtypeconfig/controller.go

```diff
@@ -289,11 +291,11 @@ func (c *Controller) startSyncController(tc *corev1a1.FederatedTypeConfig) error
 	// cluster-scoped federation control plane.  A namespace-scoped
 	// control plane would still have to use a non-shared informer due
 	// to it not being possible to limit its scope.
+	kind := tc.Spec.FederatedType.Kind
 	fedNamespaceAPIResource, err := c.getFederatedNamespaceAPIResource()
 	if err != nil {
-		return err
+		return errors.WithMessagef(err, "Unable to start sync controller for %q due to missing FederatedTypeConfig for namespaces", kind)
```

Comment:
Why is `WithMessagef` preferable in this instance to `Wrapf`?

Reply:
When I looked at the `Wrapf` API call, it seemed to want to dump a stack and felt that was unnecessary. But it looks like that's only with the `%+v` format specifier. I've fixed.